src/plugins/explore/public/components/visualizations/types.ts

```diff
@@ -263,6 +264,12 @@ export interface AxisConfig {
     milliseconds?: string;
   };
 }
+export interface ThresholdOptions {
+  thresholds?: Threshold[];
+  baseColor?: string;
+  thresholdStyle?: ThresholdMode;
+  useThresholdColor?: boolean;
```

Comment:
I'm thinking should `useThresholdColor` be in `ThresholdOptions`? It feels more like a chart-specific style options.